src/node/miner.cpp

```diff
@@ -33,6 +33,13 @@ int64_t UpdateTime(CBlockHeader* pblock, const Consensus::Params& consensusParam
     int64_t nOldTime = pblock->nTime;
     int64_t nNewTime{std::max<int64_t>(pindexPrev->GetMedianTimePast() + 1, TicksSinceEpoch<std::chrono::seconds>(NodeClock::now()))};
 
+    if (consensusParams.enforce_BIP94) {
+        const int nHeight = pindexPrev->nHeight + 1;
```

Comment:
Could you choose a different  name for this variable than `nHeight`? Aside from the outdated style `next_height` or something like that is describing it better.